cmd/dep/init.go

```diff
@@ -58,6 +58,19 @@ func trimPathPrefix(p1, p2 string) string {
 	return p1
 }
 
+func createDirIfNotExist(path string) error {
+	if _, err := os.Stat(path); err != nil {
+		if os.IsNotExist(err) {
+			if e := os.Mkdir(path, os.FileMode(0755)); e != nil {
```

Comment:
`os.MkdirAll` here will take care of creating any intermediary directories, if necessary.

Also, let's use 0777 so that it's entirely up to the umask to decide what perms are applied.